plugin/asyncrun.vim

```diff
@@ -135,6 +135,11 @@ endif
 
 if !exists('g:asyncrun_encs')
 	let g:asyncrun_encs = ''
+	if match(&shell, 'cmd') >= 0
+		let cp = system("@echo off && for /f \"tokens=2* delims=: \" %a in ('chcp') do (echo %a)")
```

Comment:
invoking `system` on windows will create a temporary console window, which sucks.
people can append this to their `_vimrc` file if they don't care.

Reply:
update to prevent console window at startup

as default config, i bet a temp console window would be better than strangely output